### PR TITLE
Retire exercise .gitignore

### PR DESCRIPTION
**Pull request by keiravillekode**

### CODE REVIEW

File: bin/create-exercise
Comment (kotp) [inline bin/create-exercise:35]: There is a reason why this was here in the first place and so removing this is likely to impact something.

Is their documentation to update that addresses what this change is going to make happen, either in the view of the student, or the view of the maintainers?

How does this change impact what happens in this project?

**Review by kotp — COMMENTED** [review]
Looks good to me, but I wonder, in terms of Chesterton's Fence...

**Review by kotp — APPROVED** [review]

**Comment by kotp** [comment]
I do not have cross-track-maintainers access any more, if I had it before, so my approval is not effective here.

**Review by BNAndras — APPROVED** [review]

File: bin/create-exercise
Comment (keiravillekode) [reply bin/create-exercise:35]: I put that there, to automate making new exercises like the existing exercises.

Bur we don't have any reason why the existing exercises had .gitignore

**Pull request merged**
